pubsub/sample.rb

```diff
@@ -0,0 +1,94 @@
+# Copyright 2015 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "gcloud"
+
+gcloud = Gcloud.new "my-gcp-project-id"
+pubsub = gcloud.pubsub
+
+pubsub.project
+
+def create_topic
+  gcloud = Gcloud.new "my-gcp-project-id"
+  pubsub = gcloud.pubsub
+
+  topic = pubsub.create_topic "my-topic"
+
+  puts "Topic created #{topic.name}"
+end
+
+def create_subscription
+  gcloud = Gcloud.new "my-gcp-project-id"
+  pubsub = gcloud.pubsub
+  topic = pubsub.topic "my-topic"
+
+  subscription = topic.subscribe "my-subscription"
+
+  puts "Subscription created #{subscription.name}"
+end
+
+def create_push_subscription
+  gcloud = Gcloud.new "my-gcp-project-id"
+  pubsub = gcloud.pubsub
+  topic = pubsub.topic "my-topic"
+
+  subscription = topic.subscribe(
+    "my-subscription-push",
+    endpoint: "https://my-gcp-project-id.appspot.com/push"
+  )
+
+  puts "Push subscription created #{subscription.name}"
+end
+
+def publish_message
+  gcloud = Gcloud.new "my-gcp-project-id"
+  pubsub = gcloud.pubsub
+  topic  = pubsub.topic "my-topic"
+
+  topic.publish "A Message"
+end
+
+def print_messages
```

Comment:
`def pull_messages`

The methods above have names that correspond to the doc samples, eg. "[Create topic](https://cloud.google.com/pubsub/publisher#create)", "[Publish messages](https://cloud.google.com/pubsub/publisher#publish)", ...

`def <Pub/Sub verb>_<Pub/Sub noun>` eg. `create_topic`, `create_subscription`, `publish_message`

The methods below here are inconsistent.  They follow `def print_<Pub/Sub noun>`.  Reading the methods above, I would expect these methods to be: `pull_messages`, `list_topics`, `list_subscriptions`.  These correspond with the docs.